gdma/default.nix

```diff
@@ -32,6 +32,8 @@ stdenv.mkDerivation rec {
 
     hardeningDisable = [ "format" ];
 
+    passthru = { inherit mpi; };
```

Comment:
`gdma` does not use MPI.

Reply:
Sorry, this one slipped in. Must have confused it with GPAW.